test/Datadog.Trace.ClrProfiler.Managed.Tests/Reflection/ModuleLookupTests.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace.ClrProfiler.Emit;
+using Xunit;
+
+namespace Datadog.Trace.ClrProfiler.Managed.Tests
+{
+    public class ModuleLookupTests
+    {
+        [Fact]
+        public void Lookup_SystemData_Succeeds_WithTwentyConcurrentTries()
+        {
+            var tasks = new Task[20];
+            var resetEvent = new ManualResetEventSlim(initialState: false);
+            var bag = new ConcurrentBag<Module>();
+            var systemDataGuid = typeof(System.Data.DataTable).Assembly.ManifestModule.ModuleVersionId;
+
+            for (var i = 0; i < 20; i++)
+            {
+                var task = new Task(() =>
```

Comment:
`Task.Run()` starts the task, no need to start is separately below.
```suggestion
                var task = Task.Run(() =>
```